lucene/spatial3d/src/test/org/apache/lucene/spatial3d/geom/Geo3DUtil.java

```diff
@@ -16,89 +16,13 @@
  */
 package org.apache.lucene.spatial3d.geom;
 
-import org.apache.lucene.geo.Polygon;
-import org.apache.lucene.geo.GeoUtils;
-
-import java.util.List;
-import java.util.ArrayList;
-
 class Geo3DUtil {
 
-  /** How many radians are in one earth surface meter */
-  final static double RADIANS_PER_METER = 1.0 / PlanetModel.WGS84_MEAN;
   /** How many radians are in one degree */
   final static double RADIANS_PER_DEGREE = Math.PI / 180.0;
   /** How many degrees in a radian */
   final static double DEGREES_PER_RADIAN = 180.0 / Math.PI;
```

Comment:
This was added because Math#toDegrees and Math#toRadians were in the forbidden APIs. I believe in master the van has been lifted (java 11) so this is not needed anymore. Not for this PR but good to have it in mind.